ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-group-du-sno-ranGen.yaml

```diff
@@ -34,24 +34,43 @@ policies:
     - path: source-crs/ClusterLogForwarder.yaml
       patches:
       - spec:
+          serviceAccount:
+            name: logcollector
           outputs:
-            - type: "kafka"
-              name: kafka-open
-              # below url is an example
-              url: tcp://10.46.55.190:9092/test
+            - name: default-lokistack
+              type: lokiStack
+              lokiStack:
+                target:
+                  name: logging-loki
+                  namespace: openshift-logging
+                authorization:
+                  token:
+                    from: serviceAccount
+              tls:
+                ca:
+                  key: service-ca.crt
+                  configMapName: openshift-service-ca.crt
           pipelines:
-          - inputRefs:
-            - audit
-            - infrastructure
-            labels:
-              label1: test1
-              label2: test2
-              label3: test3
-              label4: test4
-            name: all-to-default
+          - name: all-to-default
+            inputRefs:
+              - applications
+              - audit
+              - infrastructure
             outputRefs:
-            - kafka-open
-    - path: source-crs/ClusterLogging.yaml
+            - default-lokistack
+    - path: source-crs/LokiStack.yaml
+      patches:
+      - spec:
+          storageClassName: fs-loki-logging
+    - path: source-crs/StoragePVC.yaml
+      patches:
+      - spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 100Gi
+          storageClassName: "fs-loki-logging"
```

Comment:
No Loki or local storage of logs.

Reply:
I've dropped the storagepvc change and loki